widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -723,6 +733,29 @@ private void subscribeToEngagementStateChange() {
         );
     }
 
+    private void subscribeToOmnibrowseEvents() {
+        Glia.omnibrowse.on(Omnibrowse.Events.ENGAGEMENT, omnibrowseEngagement -> {
+            omnibrowseEngagement.getMedia().on(Media.Events.OPERATOR_STATE_UPDATE, this::onNewOperatorMediaState);
+            omnibrowseEngagement.on(Engagement.Events.STATE_UPDATE,  engagementState -> {
+                switch (engagementState.getVisitorStatus()) {
+                    case ENGAGED:
+                        onEngagementOngoing(engagementState.getOperator());
+                        break;
+                    case TRANSFERRING:
+                        Logger.e(TAG, "CallVisualizer state TRANSFERRING", new Exception("CallVisualizer flow should not have TRANSFERRING state"));
```

Comment:
NIT: think this would be a better explanation
```suggestion
                        Logger.e(TAG, "CallVisualizer state TRANSFERRING", new Exception("CallVisualizer flow does not support engagement transfers"));
```